homeassistant/components/onewire/__init__.py

```diff
@@ -41,3 +43,9 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
     if unload_ok:
         hass.data[DOMAIN].pop(config_entry.entry_id)
     return unload_ok
+
+
+async def options_update_listener(hass: HomeAssistant, entry: ConfigEntry) -> None:
+    """Handle options update."""
+    _LOGGER.info("Configuration options updated, reloading OneWire integration")
```

Comment:
Debug level max, please.

Reply:
Just to get you straight: Are you saying I should silence this by using _LOGGER.debug or similar ?

Reply:
Yes, please use debug level max for these kind of messages.